crates/sui-core/src/authority_active/gossip/mod.rs

```diff
@@ -177,97 +203,83 @@ where
             );
         };
 
-        // Return the internal result
-        let result = result.unwrap();
-        // minimum_time.await;
-        (peer_name, result)
+        (peer_name, result.unwrap())
```

Comment:
Maybe check before we unwrap, or return the result? (I know this is preexisting code)